qmlcomps/videocamera/Handle.qml

```diff
@@ -1,11 +1,11 @@
-import QtQuick
+import QtQuick 2.0
```

Comment:
why 2.0?

Reply:
it was already in the file i take directly from original repo, that as no effect in the app